javascript-1/Structura Iterativa Si Listele/script3.js

```diff
@@ -0,0 +1,27 @@
+const Marca = ["BMW", "Audi", "Dacia", "Audi", "Audi", "Dacia", "Dacia", "Dacia", "BMW", "Volkswagen", "Opel", "Mercedes", "BMW", "Volkswagen"];
```

Comment:
Ca si conventie scrierea se face cu camelCase, astfel variabilele ma astept sa fie numite ```marca```, ```pretCumparare``` in loc de ```Marca``` si ```PretCumparare```